fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -19,13 +19,17 @@ def self.run(params)
         app.all_build_train_numbers.each do |train_number|
           app.all_builds_for_train(train: train_number).each do |build|
             download_url = build.details.dsym_url
-            result = Net::HTTP.get(URI(download_url))
-            file_name = "#{app.bundle_id}-#{train_number}-#{build.build_version}.dSYM.zip"
-            File.write(file_name, result)
-            UI.success("🔑 Successfully downloaded dSYM file for #{train_number} - #{build.build_version} to '#{file_name}'")
+            if download_url =~ URI.regexp
+              result = Net::HTTP.get(URI(download_url))
```

Comment:
@KrauseFx not very important today, but if the url ever changes to be SSL, this will fail (silently...).

Maybe 
```
require "open-uri"
result = URI.parse(download_url)
```

is better ?

http://stackoverflow.com/questions/5786779/using-nethttp-get-for-an-https-url